modules/update-restricted-network-cli.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-restricted-network.adoc
+
+[id="update-restricted-network-cli_{context}"]
+= Update the restricted network cluster
+
+After you update the contents of the mirror registry, you can update your
+restricted network cluster.
+
+.Prerequisites
+
+* You have access to the mirror registry that you used to store the images that
+you used to install {product-title}.
+* You have the `ImageContentSourcePolicy` information for the content that you
+mirrored.
+* The the OpenShift Command-line Interface (CLI), commonly known as `oc`, is
+installed.
+* The `jq` package is installed.
+
+.Procedure
+
+Complete the following steps on the bastion host:
+
+. View any existing `ImageContentSourcePolicy` resources for your cluster:
++
+----
+$ oc get ImageContentSourcePolicy -o yaml
+----
+
+. If the previous command returned an `ImageContentSourcePolicy` resource,
```

Comment:
At this step, upgrade is not started yet, before your cluster is totally upgraded, I do not suggest to remove the old ones. It maybe risky.

Reply:
Yeah.  Union here.  Prune later, if you want, certainly after upgrade completes.